### DISCUSSION TITLE
[SOLVED] Can javascript be disabled in the regular version?

### DISCUSSION THREAD

**Discussion by starbuck3000** [martinrotter/rssguard#874] [Q&A] [answered]
In the documentation, it says the lite version "does not have JavaScript support". If I download the full version, is there an option to disable JavaScript in the preview pane?

**Comment by martinrotter** [comment, marked as answer]
right click preview pane there it is in submenu

**Reply by starbuck3000** [reply]
Thank you! I will switch to the full version then.
